README.markdown

```diff
@@ -194,19 +194,13 @@ Follow the instructions below if you are updating from Symphony 2.0 (not from Gi
 
 Symphony uses [Grunt](http://gruntjs.com/) to build concatenated and minified assets:
 
-1. Install `grunt-cli` globally ([see the Grunt documentation](http://gruntjs.com/getting-started)):
-
-		npm install -g grunt-cli
```

Comment:
> I think the idea behind having the cli vs the lib is that it allows you to have a specific grunt version per project

True, I did only read the manual after submitting the change. I still think it’s better not to rely on anything to be installed globally, but my original argument is no longer valid. 

Feel free to ignore this and only fix the version number.

> If we remove this, don't we need to add the grunt-cli package into package.json ?

No, for the exact same reasons you mentioned. It's only a "remote control" for the actual lib in the project. If we use the lib in the project directly, we no longer need the remote.